src/modules/users/resolvers/mentors-list.js

```diff
@@ -0,0 +1,19 @@
+const { schemaComposer } = require('graphql-compose')
+const User = require('../../../models/user')
+
+const UserTC = schemaComposer.getOTC('User')
+
+UserTC.addResolver({
+  name: 'mentors',
```

Comment:
No test was written that checks this resolver

Reply:
added test